core/math/math_funcs.h

```diff
@@ -628,24 +628,11 @@ class Math {
 	}
 
 	static _ALWAYS_INLINE_ float absf(float g) {
-		union {
-			float f;
-			uint32_t i;
-		} u;
-
-		u.f = g;
-		u.i &= 2147483647u;
-		return u.f;
+		return ::fabs(g);
```

Comment:
`::fabsf` I think does this specific to float, `::fabs` afaik is specifically for double.

Reply:
Oof, good catch! I've been using `std::fabs` so far and it works for both `float` and `double`. I naively removed `std::` because Windows compilers complained.

Reply:
Yea, that was the problem.
Makes me want to replace the `math.h` and `float.h` includes in `math_funcs.h`, but that's another rabbit hole to go down some other day...